### PR TITLE
Bootstrap alert compatible

### PR DESCRIPTION
**Pull request by saharin88**
Alert compatibility with new Bootstrap versions.

### CODE REVIEW

File: media/system/js/core-uncompressed.js [hunk 1]
```diff
@@ -324,6 +324,7 @@ Joomla.editors.instances = Joomla.editors.instances || {
 			// Message class
 			alertClass = (type == 'notice') ? 'alert-info' : 'alert-' + type;
 			alertClass = (type == 'message') ? 'alert-success' : alertClass;
+			alertClass = (type == 'error') ? 'alert-error alert-danger' : alertClass;
```
Comment (C-Lodder) [inline media/system/js/core-uncompressed.js]: Change `==` to `===`

Also apply for the lines above too.

**Comment by C-Lodder** [comment]
Please also commit the minified file

**Comment by saharin88** [comment]
#18910

**Comment by C-Lodder** [comment]
Please do it in the same PR, not a separate one

**Comment by ghost** [comment]
@saharin88 can this PR be tested? If yes, can you give Test Instructions?

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on cd53caa10754ea6f19c3b790ceb175cbcf60cf48<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18909">issues.joomla.org/tracker/joomla-cms/18909</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cd53caa10754ea6f19c3b790ceb175cbcf60cf48<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18909">issues.joomla.org/tracker/joomla-cms/18909</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18909">issues.joomla.org/tracker/joomla-cms/18909</a>.</sub>

**Comment by mbabker** [comment]
The conflicts need to be resolved here to merge.

**Comment by Quy** [comment]
How to minify a file?

**Comment by C-Lodder** [comment]
@Quy - https://skalman.github.io/UglifyJS-online/

**Pull request merged**